docker/chromium/Dockerfile

```diff
@@ -1,8 +1,26 @@
-
-FROM demisto/python3:3.7.4.1150
+FROM demisto/python3-deb:3.7.4.1163
 
 COPY requirements.txt .
+COPY google-chrome-stable_current_amd64.deb .
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  gcc \
+  python-dev \
+  wget \
+  libxss1 \
+  libappindicator1 \
+  libindicator7 \
+&& pip3 install --no-cache-dir -r requirements.txt
+
+# Install Chrome
+#RUN wget https://dl.google.co/linux/direct/google-chrome-stable_current_amd64.deb; \
+RUN dpkg -i google-chrome*.deb; \
+apt --fix-broken -y install; \
+apt-get -y install unzip
 
-RUN apk --update add --no-cache --virtual .build-dependencies python-dev build-base wget git \
-  && pip install --no-cache-dir -r requirements.txt \
-  && apk del .build-dependencies
+# Clean Up
```

Comment:
Don't do the cleanup in a separate RUN. It should be done within each separate RUN.

Reply:
if I do the clean up before the dpkg, the installation will fail

Reply:
I added a commit regarding what I meant about cleanup. You can see that now the image on docker hub is smaller.